proto/Cargo.toml

```diff
@@ -14,7 +14,7 @@ build = "build.rs"
 [dependencies]
 protobuf = "~2.0"
 futures = "0.1"
-grpcio = { path = "..", version = "0.3.0" }
+grpcio = { path = "..", version = "0.4.0" }
 
 [build-dependencies]
```

Comment:
`grpcio-compiler = { path = "../compiler", version = "0.3.0" }` below should be changed to `grpcio-compiler = { path = "../compiler", version = "0.4.0" }` I think.

Reply:
You are right!